management-model/src/main/java/org/terracotta/management/stats/NumberUnit.java

```diff
@@ -20,7 +20,7 @@
  */
 public enum NumberUnit {
 
-  PERCENT {
+  RATIO {
```

Comment:
Percent is not the english word. Stats matching this enum are called ratio in english.